tests/validator_test.go

```diff
@@ -1198,3 +1200,25 @@ func certsEqual(certs1, certs2 []*x509.Certificate) bool {
 	}
 	return true
 }
+
+func retrieveNodeArchitecture() string {
+	nodeList := &core.NodeList{}
+	err := apiClient.List(ctx, nodeList)
+	Expect(err).NotTo(HaveOccurred(), "Failed to list nodes")
+	nodes := nodeList.Items
+	Expect(nodes).NotTo(BeEmpty(), "No nodes found")
+	architecture := nodes[0].Status.NodeInfo.Architecture
+	Expect(architecture).To(BeElementOf([]string{"s390x", "amd64"}), "Unsupported architecture")
```

Comment:
What about ARM?

Reply:
We could add support for ARM and/or AArch64. According to this [link](https://people.redhat.com/~cohuck/2022/01/05/qemu-machine-types.html), it should correspond to the virt QEMU machine type. Do you have machines available for testing?